Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -123,6 +125,19 @@ final class StatusTableViewController: LoopChartsTableViewController {
         addScenarioStepGestureRecognizers()
 
         tableView.backgroundColor = .secondarySystemBackground
+    
+        tableView.register(TempStatusCell.self, forCellReuseIdentifier: TempStatusCell.className)
+        notificationsCriticalAlertPermissionsViewModel.$showWarning
+            .receive(on: RunLoop.main)
+            .sink { [weak self] in
+                let current = self?.tableView.numberOfRows(inSection: Section.tempStatus.rawValue) ?? 0
```

Comment:
[nit] Consider `guard let self = self else { return }`.

Reply:
Consider `let visible = (self.tableView.numberOfRows(inSection: Section.tempStatus.rawValue) ?? 0) != 0` as more clear about what you are checking. Then change conditionals to `if !showWarning && visible {} else if showWarning && !visible {}`. I think this is more understandable.